### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by chandru-tkc**
Updated documentation to clarify prismEndpoint for users.

### CODE REVIEW

File: charts/nutanix-csi-storage/README.md [hunk 1]
```diff
@@ -161,12 +161,15 @@ Install the driver in the `ntnx-system` namespace and create a volume storagecla
 ```console
 helm install nutanix-storage nutanix/nutanix-csi-storage -n ntnx-system --create-namespace --set volumeClass=true --set prismEndPoint=X.X.X.X --set username=admin --set password=xxxxxxxxx --set storageContainer=container_name --set fsType=xfs
 ```
+In the above command  `prismEndpoint` refers to the Prism Element cluster virtual ip address where k8s vms are running.
```
Comment (tuxtof) [inline charts/nutanix-csi-storage/README.md:164]: agree for the first part to clarify we are talking about Prism Element cluster virtual ip, but k8s vm can run anywhere not only on this cluster so let's replace by something like

`In the above command prismEndpoint refers to the Prism Element cluster virtual ip address where storage will be consumed.`

File: charts/nutanix-csi-storage/README.md [hunk 2]
```diff
@@ -161,12 +161,15 @@ Install the driver in the `ntnx-system` namespace and create a volume storagecla
...
 ```
+In the above command  `prismEndpoint` refers to the Prism Element cluster virtual ip address where k8s vms are running. 
+
 
 Install the driver in the `ntnx-system` namespace, create a volume and a dynamic file storageclass and set the volume storage class as default:
 
 ```console
 helm install nutanix-storage nutanix/nutanix-csi-storage -n ntnx-system --create-namespace --set volumeClass=true --set prismEndPoint=X.X.X.X --set username=admin --set password=xxxxxxxxx --set storageContainer=container_name --set fsType=xfs --set defaultStorageClass=volume --set dynamicFileClass=true --set fileServerName=name_of_the_file_server
 ```
+In the above command  `prismEndpoint` refers to the Prism Element cluster virtual ip address where k8s vms are running.
```
Comment (tuxtof) [inline charts/nutanix-csi-storage/README.md:172]: same as above

**Review by tuxtof — CHANGES_REQUESTED** [review]

File: charts/nutanix-csi-storage/README.md (hunk 2 above)
Comment (chandru-tkc) [reply charts/nutanix-csi-storage/README.md:172]: Sure thanks Chris, will update it

**Comment by chandru-tkc** [comment]
Updated the documentation with correct statement

**Review by tuxtof — APPROVED** [review]

**Pull request merged**